### PR TITLE
DOCS-3019 Adds iOS Web View Tracking Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What and why?

Adds iOS Web View Tracking documentation.

### How?

DOCS-3019

### Additional Notes

Preview link in Documentation PR: https://docs-staging.datadoghq.com/alai97/single-sourcing-web-view-tracking-docs/real_user_monitoring/ios/web_view_tracking/.

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [ ] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing change.

### CODE REVIEW

File: docs/rum_collection/web_view_tracking.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "Browser Monitoring"
+---
+
+## Overview
+
+Real User Monitoring allows you to monitor web views and eliminate blind spots in your hybrid iOS and tvOS applications.
+
+You can:
```
Comment (urseberry) [inline docs/rum_collection/web_view_tracking.md:21]:
```suggestion
You can perform the following:
```

**Review by urseberry — APPROVED** [review]

File: docs/rum_collection/web_view_tracking.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+   // Start tracking
+   webView.configuration.userContentController.trackDatadogEvents(in: hosts)
+   // Stop tracking
+   // Note: This method must be called when the webview is de-initialized.
+   webView.configuration.userContentController.stopTrackingDatadogEvents()
+   ```
+
+## Access your web views
+
+Your web views appear as events and views in the [RUM Explorer][4]. Filter on your iOS and tvOS applications and click a session. A side panel with a list of events in the session appears.
```
Comment (urseberry) [inline docs/rum_collection/web_view_tracking.md:49]:
```suggestion
Your web views appear as events and views in the [RUM Explorer][4]. Filter on your iOS and tvOS applications, and click a session. A side panel with a list of events in the session appears. 
```

File: docs/rum_collection/web_view_tracking.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+2. Edit your existing iOS SDK setup from [RUM iOS Monitoring][3].
+3. Add tracking for web views with the following example:
+
+   ```
+   // Start tracking
+   webView.configuration.userContentController.trackDatadogEvents(in: hosts)
+   // Stop tracking
+   // Note: This method must be called when the webview is de-initialized.
+   webView.configuration.userContentController.stopTrackingDatadogEvents()
+   ```
```
Comment (ncreated) [inline docs/rum_collection/web_view_tracking.md:45]: Unlike in Android SDK, the iOS APIs for webview tracking are not part of the SDK setup which we link to in `[3]`. Instead, we provide the two APIs from the snippet. IMO, we should change this section from _"Update your existing SDK setup"_, to something like _"Instrument web view with Datadog"_. We could use markdown to list and explain each API like we do [here](https://docs.datadoghq.com/real_user_monitoring/ios/advanced_configuration/?tab=swift#rum-configuration).

I can propose something like this 🙂:

> ### Instrument your web views
> 
> The latest version of the RUM iOS SDK comes with two APIs, which let you control web view tracking. Both are declared as an extension to [`WKUserContentController`](https://developer.apple.com/documentation/webkit/wkusercontentcontroller):
>
> `trackDatadogEvents(in hosts: Set<String>)`
> : Enables tracking RUM events in this web view for particular `hosts`.
>
> `stopTrackingDatadogEvents()`
> : Disables tracking for this web view. This should be typically called when you are done with given web view and it is about to be deallocated.
>
> Example:
> ```swift
> import WebKit
> import Datadog
>  
> webView.configuration.userContentController.trackDatadogEvents(in: ["example.com"])
> ```

WDYT?

**Review by ncreated — CHANGES_REQUESTED** [review]
I left feedback on _"Update your existing SDK setup"_ - let me know WDYT 🙂.

**Review by ncreated — APPROVED** [review]
💪 looks great! As this feature is already available in `1.10.0` (GA), maybe we can rebase this onto `master` before merge?

**Pull request merged**